tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -12,18 +12,22 @@ script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 readonly repo_root_dir="$(cd "${script_dir}/../../../.." && pwd)"
 readonly kernel_builder="${repo_root_dir}/tools/packaging/kernel/build-kernel.sh"
 
+source "${script_dir}/../../scripts/lib.sh"
 
 GO_VERSION=${GO_VERSION}
 RUST_VERSION=${RUST_VERSION}
 
 DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
-container_image="shim-v2-builder"
+container_image="${CC_BUILDER_REGISTRY}:shim-v2-${GO_VERSION}-${RUST_VERSION}-$(get_last_modification ${script_dir}/Dockerfile)"
```

Comment:
Good point. It is not a change on dockerfile that will require a re-build of the image.

Nit: maybe prepend ${GO_VERSION} with `go` and `rust` likewise, i.e., `shim-v2-go${GO_VERSION}-rust${RUST_VERSION}`

Reply:
One easy way to avoid that is checking for any change in the folder where the files live.  That would cover this case.

Reply:
It is a good solution.